.mvn/jvm.config

```diff
@@ -0,0 +1,4 @@
+-Dmaven.wagon.httpconnectionManager.ttlSeconds=120
+-Dmaven.wagon.http.retryHandler.requestSentEnabled=true
+-Dmaven.wagon.http.retryHandler.count=10
+-Xmx5g
```

Comment:
why is this set?

Reply:
I planned to use same settings as Quarkus does, but it seems like they only needs this to fix transient issues on Azure, so we should be fine. I dropped it.